admin/test/features/upload_view_forms.feature

```diff
@@ -0,0 +1,28 @@
+Feature: Upload and View Forms
+
+  Background:
+    Given I have signed in with test-developer
+    And I am on the Upload and View forms page
+
+  Scenario: Upload and view forms page has a title
+    Then I should see a heading on Upload and View forms page
+
+  Scenario: Upload and view forms page has a option to upload new form
+    Then I should have the option to upload new form
+
+  Scenario: Upload new forms page has a title
+    And I am on the Upload new forms page
+    Then I should see a heading on Upload new form page
+    And I can see sub heading 'Download example' on Upload new form Page
+    And I can see sub heading 'Upload file(s)' on Upload new form Page
```

Comment:
@sinhaamit1980 Please try and keep to a 1 or 2 assertions per scenario. This looks like it has 4, which is not good as if the first one fails we have no feedback on the others, split them in to their own scenarios